src/screens/SignedChatScreen/index.tsx

```diff
@@ -129,6 +127,7 @@ const SignedChatScreen = () => {
         </View>
         <Loading visible={loading} />
       </View>
+      <Loading visible={loading} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

An extra `<Loading>` component has been appended at the end of the component. This might be a mistake or redundancy if the loading state is already being handled by the previous `<Loading>` component. If it's intentional, please ensure that having two loading components doesn't conflict or cause unexpected behavior.
```diff
-      <Loading visible={loading} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
done